chapters/descriptorsets.txt

```diff
@@ -2584,6 +2584,9 @@ ifdef::VK_VERSION_1_1,VK_KHR_maintenance1[]
     pname:imageView must: not be 2D or 2D array image view created from a 3D
     image
 endif::VK_VERSION_1_1,VK_KHR_maintenance1[]
+  * If pname:imageView is a depth/stencil image the aspectMask used to create
```

Comment:
Should read "If pname:imageView has a depth/stencil format" or "If pname:imageView is created from a depth/stencil image"